modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,185 @@
 
 Query is a service that retrieves traces from storage and hosts the user interface to display them.
 
-.Parameters used by the {TempoOperator} to define Query
+.Configuration parameters for the Querier
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    replicas:
-|Specifies the number of Query replicas to create.
-|Positive integer
+|`nodeSelector`
+|NodeSelector is the simplest recommended form of node selection constraint.
+|type: object, additionalProperties: type: string
 |
+
+|`replicas`
+| Replicas represents the number of replicas to create for this component.
+| type: integer, format: int32
+|
+
+|`tolerations`
+|Tolerations defines component specific pod tolerations.
+|type: array, x-kubernetes-list-type: atomic
+|
+
+|`Tolerations` (items)
+|The pod this Toleration is attached to tolerates any taint that matches the triple <key, value, effect> using the matching operator <operator>.
+|type: object
+|
+
+|`effect`
+| Effect indicates the taint effect to match. Empty means match all taint effects. When specified, allowed values are NoSchedule, PreferNoSchedule, and NoExecute.
+|type: string
+|
+
+|`key`
+|Key is the taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, operator must be Exists; this combination means to match all values and all keys.
+|type: string
+|
+
+|`operator`
+|Operator represents a key's relationship to the value. Valid operators are Exists and Equal. Defaults to Equal. Exists is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
```

Comment:
```suggestion
|Operator represents a key's relationship to the value for a toleration. Valid operators are Exists and Equal. Defaults to Equal. Exists is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
```

Reply:
Sounds good. What do you think about removing all toleration properties (key, value, effect and operator) and adding a link from the `tolerations` property (the list of tolerations) to https://docs.openshift.com/container-platform/4.14/nodes/scheduling/nodes-scheduler-taints-tolerations.html for an in-depth explanation of them?

Reply:
Thank you for the suggestion. Unfortunately, the repo rules don't allow me to link there from inside a module. However, WDYT (WIP).

Reply:
Yes, sounds good to me. We can probably add a line to refer to the additional resources.